client/src/components/core/ms-modal/FolderSelectionModal.vue

```diff
@@ -3,73 +3,87 @@
 <template>
   <ms-modal
     :title="title"
-    :subtitle="subtitle"
     :close-button="{ visible: true }"
     :cancel-button="{
       label: $t('TextInputModal.cancel'),
       disabled: false,
       onClick: cancel,
     }"
     :confirm-button="{
-      label: $t('TextInputModal.ok'),
+      label: $t('TextInputModal.moveHere'),
       disabled: selectedPath === startingPath,
       onClick: confirm,
     }"
   >
-    <header-breadcrumbs
-      :path-nodes="headerPath"
-      @change="onPathChange"
-    />
-    <div>
-      <ion-list>
-        <ion-list-header
-          class="folder-list-header"
-          lines="full"
+    <div class="navigation">
+      <ion-buttons>
+        <ion-button
```

Comment:
Those two buttons are completly wrong right now. Either there're getting fixed in this PR, in which case the PR should be a draft, or their logic should be removed and the buttons should be hidden.

Reply:
As mention in the description of the PR, this issue will be fixed in https://github.com/Scille/parsec-cloud/issues/6056